<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnownFilter</class>
 <widget class="QWidget" name="UnownFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">UnownFilter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>543</width>
        <height>150</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="4">
        <widget class="IVFilter" name="ivFilter" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelAbility">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ability</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="ComboBox" name="comboBoxAbility">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labelLetter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Letter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="CheckList" name="checkListLetter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="pushButtonLetterAny">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Any</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelGender">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ComboBox" name="comboBoxGender">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">♂</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">♀</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="labelHiddenPower">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Hidden Power</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="CheckList" name="checkListHiddenPower">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="pushButtonHiddenPowerAny">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Any</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelGenderRatio">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Gender Ratio</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="ComboBox" name="comboBoxGenderRatio">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Genderless</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">50% ♂ / 50% ♀</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">25% ♂ / 75% ♀</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">75% ♂ / 25% ♀</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">87.5% ♂ / 12.5% ♀</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">100% ♂</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">100% ♀</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="labelNature">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Nature</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="CheckList" name="checkListNature">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="pushButtonNatureAny">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Any</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelShiny">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Shiny</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="ComboBox" name="comboBoxShiny">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Star</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Square</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Star/Square</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="checkBoxUseDelay">
         <property name="text">
          <string>Use Delay</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="TextBox" name="textBoxDelay">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QCheckBox" name="checkBoxDisableFilters">
         <property name="text">
          <string>Disable Filters</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>CheckList</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/CheckList.hpp</header>
   <slots>
    <slot>resetChecks()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>IVFilter</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/IVFilter.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>comboBoxAbility</tabstop>
  <tabstop>comboBoxGender</tabstop>
  <tabstop>comboBoxGenderRatio</tabstop>
  <tabstop>comboBoxShiny</tabstop>
  <tabstop>checkListLetter</tabstop>
  <tabstop>pushButtonLetterAny</tabstop>
  <tabstop>checkListHiddenPower</tabstop>
  <tabstop>pushButtonHiddenPowerAny</tabstop>
  <tabstop>checkListNature</tabstop>
  <tabstop>pushButtonNatureAny</tabstop>
  <tabstop>checkBoxUseDelay</tabstop>
  <tabstop>textBoxDelay</tabstop>
  <tabstop>checkBoxDisableFilters</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
